javaslang/src/main/java/javaslang/collection/PriorityQueue.java

```diff
@@ -24,18 +24,20 @@
 
     private final SerializableComparator<? super T> comparator;
     private final List<Node<Ranked<T>>> forest;
+    private final int size;
```

Comment:
making size calculation constant was pretty easy, `insert` adds one, `merge` is the sum of the two and `tail` subtracts one :)